projects/packages/publicize/src/class-publicize-script-data.php

```diff
@@ -58,6 +58,11 @@ public static function set_admin_script_data( $data ) {
 
 		if ( empty( $data['site']['plan']['product_slug'] ) ) {
 			$data['site']['plan'] = Current_Plan::get();
+
+			// Override the features for Simple sites.
+			if ( ( new Host() )->is_wpcom_simple() ) {
+				$data['site']['plan']['features'] = Current_Plan::get_simple_site_specific_features();
```

Comment:
Will be good to sandbox this and see how many times the function `get_simple_site_specific_features` gets hit. Will that lead to an increase in DB calls?

Reply:
That function call is already cached. You can see the method in this PR.